base/lttagging.dtx

```diff
@@ -209,6 +219,19 @@
          \ERRORusetaggingsocket  % that should get a proper error message
 }
 %    \end{macrocode}
+%    The same as an expandable command:
+%    \begin{macrocode}
+\cs_new:Npn \UseExpandableTaggingSocket #1 {
+  \int_case:nnF
+      { \int_use:c { c__socket_tagsupport/#1_args_int } }
+         {
+           0 \prg_do_nothing:
+           1 \use_none:n
+           2 \use_ii:nn
+         }
+         \ERRORusetaggingsocket  % that should get a proper error message
+}
+%    \end{macrocode}
```

Comment:
Do we need expandable tagging sockets with arguments? (There is only one l3layer version without argument). And should there be an error condition? If not the definition could be simplified to `\cs_new:Npn \UseExpandableTaggingSocket #1 {\prg_do_nothing:}`

Reply:
Currently I do not think that we have a need for them. I don't think that the command should be simplified like that though since then attempts to use them would silently do the wrong thing. Instead they should either work or have a clear error. Given that it's not really more work to just support the case here and it's more consistent, I would support arguments in `\UseExpandableTaggingSocket`, but I also wouldn't mind changing this to error instead.

Reply:
I don't mind either way, only asking. But in any case we need underlying expandable sockets in ltsockets, as with tagging active I must be able to call the socket really.

Reply:
I think keeping them aligned as suggested by @zauguin is fine (even if we don't use them right now. We clear do not intend to provide a different  behavior other than one is epeandabel and the other is not.